Adafruit_MQTT.h

```diff
@@ -257,8 +257,9 @@ class Adafruit_MQTT {
   // Functions to generate MQTT packets.
   uint8_t connectPacket(uint8_t *packet);
   uint8_t disconnectPacket(uint8_t *packet);
+  static uint16_t packetAdditionalLen(uint16_t currLen);
   uint16_t publishPacket(uint8_t *packet, const char *topic, uint8_t *payload,
-                         uint16_t bLen, uint8_t qos);
+                         uint16_t bLen, uint8_t qos, uint16_t maxPacketLen);
```

Comment:
@flavio-fernandes  is there a way to protect against memory corruption without changing the function signature here?

Reply:
Not really. 😄   I think that the only way we can protect from the memory corruption is by knowing how much space is available to begin with.
This is a private member function, so there should not be a concern about changing its signature, agree?
Regardless, I made the extra parameter have the [default value](https://www.w3schools.com/cpp/cpp_function_default.asp) of 0, so caller will still work should we try to use the old function arguments.